server/services/caseloadService.ts

```diff
@@ -15,6 +22,16 @@ export default class CaseloadService {
       .filter(offender => offender.nomsNumber)
       .map(offender => offender.nomsNumber)
 
+    /*
+     TODO: Maybe this should be checking for existing licences by nomisId rather than staffId? What if the offender
+      changes their managing officer after a licence has been created for them?
+     */
+    const existingLicences = await this.licenceService.getLicencesByStaffIdAndStatus(staffIdentifier, username, [
```

Comment:
I agree it should use nomisId - or a list of nomisId - to get all licences in officer's caseload? 
We need to think about what happens if the staff details on the licence is now different to the staff user who has this offender in their caseload. Its a suggestion that the offender has been reassigned to a different probation officer - so should we also update the details on the licence to match the current user?  OK for now though - this needs to be discussed as a different ticket.